content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Moving Local Experiments to the Cloud with Terraform Provider Iterative
```

Comment:
Wait so is "Terraform Provider Iterative" the official long name of the tool? I realize that's what TPI seems to stand for but once expanded it's not that readable.

In https://registry.terraform.io/providers/iterative/iterative/latest/docs it's just called "Iterative Provider" (for Terraform)

Reply:
p.s. I see "Iterative's Terraform Provider" is used in the description, which is better too.

Reply:
"Terraform Provider Iterative" is the official name due to Terraform's exceedingly strict naming conventions. We don't have a choice :/